File: Java/1450_Number_of_Students_Doing_Homework_at_a_Given_Time.java
class Solution {
    public int busyStudent(int[] startTime, int[] endTime, int queryTime) {
        int result = 0;
        for(int i = 0; i < startTime.length; i++){
            result += (queryTime >= startTime[i] && queryTime <= endTime[i]) ? 1 : 0;
        }
        return result;
    }
}

File: CS/1450_Number_of_Students_Doing_Homework_at_a_Given_Time.cs
public class Solution {
    public int BusyStudent(int[] startTime, int[] endTime, int queryTime) {
        int result = 0;
        for(int i = 0; i < startTime.Length; i++){
            if((startTime[i] <= queryTime) && (queryTime <= endTime[i])){
                result++;
            }
        }
        return result;
    }
}
